applications/dashboard/controllers/api/UsersApiController.php

```diff
@@ -318,6 +355,48 @@ public function post(array $body) {
         return new Data($result, 201);
     }
 
+    /**
+     * Set a new photo on a user.
+     *
+     * @param $id A valid user ID.
+     * @param array $body The request body.
+     * @throws ClientException if the image provided is not supported.
+     * @return array
+     */
+    public function post_photo($id = null, array $body) {
+        $this->permission('Garden.SignIn.Allow');
+
+        $photoUploadSchema = UploadedFileSchema::parse([]);
```

Comment:
It would be more efficient to use `new UploadedFileSchema();` since Schema::parse() does unnecessary operations.
Also, UploadedFileSchema constructor does not take any parameters so supplying an empty array will just confuse anyone reading that code.